GettingStarted.md

```diff
@@ -1,12 +1,176 @@
 # Getting Started with Localytics in Xamarin
 ---
-This Xamarin Component simplifies the process of integrating Localytics into Xamarin iOS, Android and Forms projects.
+This Xamarin Nuget simplifies the process of integrating Localytics into Xamarin iOS, Android and Forms projects.
 
 The steps to integrate Localytics with this component mirrors the native Localytics SDK, so the following docs for the respective platforms maybe helpful:
 
 * [iOS Localytics Docs](http://docs.localytics.com/dev/ios.html)
 * [Android Localytics Docs](http://docs.localytics.com/dev/android.html)
 
+
+## Getting Started
+
+### Install the SDK
+1. Download the latest version of the Nuget package from [here](https://downloads.localytics.com/SDKs/Xamarin/Localytics-Xamarin-Latest.zip).
+2. Add a Nuget Package source in Visual Studio Preferences called localytics that points to the folder with the downloaded package.
+3. In Projects where you want to use Localytic SDK 
+4. Integrating to each of your projects
+	* Expand Packages folder in your platform Project
+	* Add a Package
+	* Select localytics as the package sources
+	* Pick the latest version of localytics
+	* Repeat the step for each platform (Android and iOS)
+	
+## Package Names
+
+* LocalyticsXamarin.iOS - API's available for using in the iOS platform from the Native SDK [iOS Localytics Docs](http://docs.localytics.com/dev/ios.html)
+* LocalyticsXamarin.Android - API's available for use on the Android platform [Android Localytics Docs](http://docs.localytics.com/dev/android.html)
+* LocalyticsXamarin.Shared - Platform specific API's that have been bridged to work from a Shared library project.
+* LocalyticsXamarin.Common - class available on Both iOS and Android using .net dependencies only. The Interfaces are the same across platforms offering similar functionality.
+
+## Xamarin Forms and Dependency Service
+LocalyticsXamarin.Common.ILocalytics interface is implemented using XamarinForms and can be available to a project using XamarinDependency Service.
+
+Add a file to each of your platform with the following contents
+
+    using System;
+    using LocalyticsXamarin.Common;
+    [assembly: Xamarin.Forms.Dependency(typeof(LocalyticsXamarin.Shared.LocalyticsXamarinForms))]
+    namespace LocalyticsXamarin.Shared
+    {
+    public class LocalyticsXamarinForms : LocalyticsSDK, ILocalytics, IPlatform
+	    {
+	    }
+    }
+
+Use the API in common modules using the interface
+```
+    ILocalytics localytics = DependencyService.Get<ILocalytics>();
+```
+
+* Non forms classes can use the same interface using
+```
+    ILocalytics localytics = LocalyticsSDK.SharedInstance;
+``` 
+
+## API Summary
+
+API's follow c# naming conventions and are translation of the API on the native platform.
```

Comment:
nit: All API's are a translation of the API's available to native platforms and follow the C# naming convention